roxie/ccd/ccdserver.cpp

```diff
@@ -13471,6 +13670,21 @@ class CRoxieServerThreadedConcatActivity : public CRoxieServerActivity
         CRoxieServerActivity::stop();
     }
 
+    virtual RoxieSourceCharacteristics getAllInputsCharacteristics() const override
```

Comment:
Should the doStart code be changed to use startInputs?  Would there be any advantage?